src/session/session.controller.ts

```diff
@@ -0,0 +1,117 @@
+import { ParseMongoIdPipe } from '@/pipes/mongo-id.pipe';
+import { SessionSchema } from '@/schema/session.schema';
+import { CreateSessionDto } from '@/session/dto/create-session.dto';
+import { SessionService } from '@/session/session.service';
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpCode,
+  HttpException,
+  HttpStatus,
+  Param,
+  ParseIntPipe,
+  Patch,
+  Post,
+  Query,
+} from '@nestjs/common';
+import { ApiParam, ApiQuery, ApiResponse, ApiTags } from '@nestjs/swagger';
+import { MongooseError, Types } from 'mongoose';
+
+@Controller('sessions')
+@ApiTags('sessions')
+export class SessionController {
+  constructor(private readonly sessionService: SessionService) {}
+
+  @ApiQuery({ name: 'finished', type: Boolean, required: false })
+  @ApiResponse({
+    status: HttpStatus.OK,
+    description: 'Sessions list',
+    type: () => SessionSchema,
+    isArray: true,
+  })
+  @Get()
+  getSessions(@Query('finished') finished?: boolean) {
+    return this.sessionService.getSessions(finished);
+  }
+
+  @ApiResponse({
+    status: HttpStatus.OK,
+    description: 'Session',
+    type: () => SessionSchema,
+  })
+  @ApiResponse({
+    status: HttpStatus.NOT_FOUND,
+    description: 'Session not found',
+  })
+  @ApiParam({ name: 'id', type: Number, description: 'Table ID' })
+  @Get('table/:id')
+  async getSessionByTable(@Param('id', new ParseIntPipe()) id: number) {
+    const doc = await this.sessionService.getSessionByTable(id);
+    if (!doc) {
+      throw new HttpException('Session not found', HttpStatus.NOT_FOUND);
```

Comment:
Using the message "Session not found" when getting it from the table is not right. Maybe use something fitter like "No session found in the table" or "There are no sessions in this table" idk